python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -320,8 +336,8 @@ def update_directions(db_manager: DatabaseManager) -> None:
         )
         .select_from(VehicleTrips)
         .join(
-            TempEventCompare,
-            TempEventCompare.pm_trip_id == VehicleTrips.pm_trip_id,
+            temp_trips,
+            temp_trips.c.pm_trip_id == VehicleTrips.pm_trip_id,
```

Comment:
was there an index update that made this subquery better than using the TempEventCompare table directly?

Reply:
I actually think this can be re-written. We don't need to join to the `VehicleTrips` table in this subquery. I'll update this.